### PR TITLE
feat: instructions to perform root-user actions

### PR DESCRIPTION
**Pull request by Reb-B**

### CODE REVIEW

File: content/labs/perform-root-user-actions-in-subaccounts/_index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+In some cases you will need to perform an action in a subaccount that can only be done by a root-user. For example. this is necessary if you want to change your support plan, enable multifactor authentication, or close the account. For a longer list of root-user actions see [this](https://docs.aws.amazon.com/general/latest/gr/root-vs-iam.html) documentation.
+
+In this lab you will learn the steps you need to do to complete such actions.
+
+
+1. Navigate to your AWS Root Account.
+1. Search for AWS Organizations in the search field at the top of the screen and select it when it appears.
+1. In AWS Organizations you will see your multi-account structure. Click on 'List'.
+1. Look for the account in which you want to perform a root user action and copy and paste the email address associated with the account.
+1. Navigate to the AWS login page and paste the email address that you copied into the email field.
```
Comment (Reb-B) [inline content/labs/perform-root-user-actions-in-subaccounts/_index.md:19]: Open an incognito window in your browser to login.

File: content/labs/perform-root-user-actions-in-subaccounts/_index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+1. You don't have a password, so click on "forgotten password".
```
Comment (Reb-B) [inline content/labs/perform-root-user-actions-in-subaccounts/_index.md:20]: accounts created via organisations don't have a root user password

File: content/labs/perform-root-user-actions-in-subaccounts/_index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+1. Fill in the captcha.
+1. The password reset url has been sent to the AWS Systems Manager. So go back to your root account.
+1. Navigate to the AWS Systems Manager and click on Parameter Store in the menu to the left.
+1. Here you will find a parameter named [ETWAS], this is the password reset url you can use to change your password and log in to your account.
```
Comment (Reb-B) [inline content/labs/perform-root-user-actions-in-subaccounts/_index.md:24]: copy the url and use it in the incognito window

File: content/labs/perform-root-user-actions-in-subaccounts/_index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+1. Reset your password using the password reset url.
```
Comment (Reb-B) [inline content/labs/perform-root-user-actions-in-subaccounts/_index.md:25]: choose a strong password preferably with your preferred password manager

**Comment by s0enke** [comment]
rebased and merge coflicts solved.
ready to merge for me when #28 is included here @Reb-B

**Pull request merged**